build.gradle.kts

```diff
@@ -15,28 +14,30 @@ buildscript {
 
 
 plugins {
-    id("io.gitlab.arturbosch.detekt") version "1.0.0.RC6-4"
+    id("io.gitlab.arturbosch.detekt") version "1.17.1"
+    id("com.github.ben-manes.versions") version "0.39.0"
```

Comment:
Allows to check if there are any dependency updates

Reply:
Don't forget to add `rejectVersionIf `

```
tasks.named("dependencyUpdates").configure {
  rejectVersionIf {
    isNonStable(it.candidate.version) && !isNonStable(it.currentVersion)
  }
}
```

Because latest version can be RC, alpha and etc. and reports can contains false positive updates.

Reply:
done